src/App.js

```diff
@@ -12,11 +12,6 @@ import { CreateUserForm } from './components/CreateUserForm';
 import UserProfile from './components/UserProfile';
 import EditProduct from './components/EditProduct';
 import UserCart from './components/UserCart';
-import Armor from './components/Armor';
-import Weapon from './components/Weapon';
-import Potion from './components/Potion';
-import ClassProducts from './components/ClassProducts';
-import CreateProduct from './components/CreateProduct';
 
```

Comment:
Should we leave CreateProduct as an import in App.js? 
Is this the thing that Thom made for the admin feature?

Reply:
I took a look at Thom's most current PR and it looks like he is relying on this import for CreateProduct. I think his merging his branch will resolve this. So I'm going to go ahead and merge this branch into dev.